travis/main.sh

```diff
@@ -94,7 +97,7 @@ cmake_args=(
 )
 
 # libxpm-dev is unavaible in brew repo
```

Comment:
```suggestion
# libxpm-dev is available in brew repo
```

Reply:
while "unavaible" is misspelled, it should not be changed to "available" here

Reply:
Oops. Sorry copy and paste error.

Reply:
perhaps a nice case for a little separate PR?